search/search_service/config.py

```diff
@@ -15,7 +15,8 @@
 PROXY_CLIENT_KEY = 'PROXY_CLIENT_KEY'
 PROXY_CLIENTS = {
     'ELASTICSEARCH': 'search_service.proxy.elasticsearch.ElasticsearchProxy',
-    'ELASTICSEARCH_V2': 'search_service.proxy.es_search_proxy.ElasticsearchProxy'
+    'ELASTICSEARCH_V2': 'search_service.proxy.es_proxy_v2.ElasticsearchProxyV2',
+    'ELASTICSEARCH_V3': 'search_service.proxy.es_proxy_v3.ElasticsearchProxyV3'
```

Comment:
idea: instead of V3, how about "V2.1" or "V2_1". I was getting anxious about calling this V3 while search is called V2 and having to explain that V2 is the API version that both V2 and V3 proxies are compatible with, although the V3 proxy is only compatible with the "version: 2" mapping. 

Doing semver takes the edge off, IMO. Makes it clear that 2.1 is a more minor evolution of whatever happened in 2.